testing/functional/tests/cypress/integration/01-student/04-application-page.spec.ts

```diff
@@ -325,81 +325,89 @@ describe("Application Page", () => {
-    applicationObject.nextSectionButton().click();
-    cy.wait("@programYear");
-    applicationObject.personalInformationButton().click();
-    applicationObject.iConfirmMyStudentAidCheckbox().click();
-    applicationObject.citizenStatusRadioButton().click();
-    applicationObject.residentOfBCRadioButton().click();
-    applicationObject.residenceNoneOfTheAboveRadioButton().click();
-    applicationObject
-      .explainSituationInputField()
-      .type("I applied but haven't received confirmation yet.");
-    applicationObject.indigenousPersonRadioButton().click();
-    applicationObject.aboriginalRadioButton().click();
-    applicationObject.legalGuardianRadioButton().click();
-    applicationObject.courseStudyStartDateRadioButton().click();
-    applicationObject.whenDidYouGraduateInputText().type("2023-12-01");
-    applicationObject.workingFullTimeRadioButton().click();
-    applicationObject.fullTimeLaborForceRadioButton().click();
-    applicationObject.allowTrustContactRadioButton().click();
-    applicationObject.nextSectionButton().click();
-    applicationObject.marriedRadioButton().click();
-    applicationObject.dateOfMarriage().type("2023-12-01");
-    applicationObject.dependRadioButton().click();
-    applicationObject.fullNameText().type("Jason Holder");
-    applicationObject.dateOfBirth().type("2003-07-01");
-    applicationObject.attendingPostSecondarySchoolRadioButton().click();
-    applicationObject.declaredOnTaxesRadioButton().click();
-    applicationObject.addAnotherDependantButton().click();
-    applicationObject.secondCloseButton().click();
-    applicationObject.doYouHaveDependentSupportRadioButton().click();
-  });
+  it(
+    "Check that all fields on the Family Information page are working.",
+    { retries: 4 },
+    () => {
+      cy.intercept("GET", "**/program-year").as("programYear");
+      applicationObject.applicationButton().should("be.visible").click();
+      applicationObject.draftApplication().eq(0).click();
+      applicationObject.draftApplicationVerifyText().should("be.visible");
+      applicationObject.nextSectionButton().click();
+      cy.wait("@programYear");
+      applicationObject.personalInformationButton().click();
+      applicationObject.iConfirmMyStudentAidCheckbox().click();
+      applicationObject.citizenStatusRadioButton().click();
+      applicationObject.residentOfBCRadioButton().click();
+      applicationObject.residenceNoneOfTheAboveRadioButton().click();
+      applicationObject
+        .explainSituationInputField()
+        .type("I applied but haven't received confirmation yet.");
+      applicationObject.indigenousPersonRadioButton().click();
+      applicationObject.aboriginalRadioButton().click();
+      applicationObject.legalGuardianRadioButton().click();
+      applicationObject.courseStudyStartDateRadioButton().click();
+      applicationObject.whenDidYouGraduateInputText().type("2023-12-01");
```

Comment:
Are these hardcoded values planned to come from a `testData` source?

Reply:
No more hardcoded values now, updated & added to fixtures.